libraries/core_libs/consensus/include/pbft/round_face.hpp

```diff
@@ -0,0 +1,82 @@
+#pragma once
+
+#include <cassert>
+#include <memory>
+
+#include "common/types.hpp"
+#include "network/network.hpp"
+#include "pbft/node_face.hpp"
+#include "vote_manager/vote_manager.hpp"
+
+namespace taraxa {
+
+const uint64_t kMaxLambda = 60 * 1000;  // in ms, max lambda is 1 minute
+#define MAX_STEPS 13                    // Need to be a odd number
+#define MAX_WAIT_FOR_SOFT_VOTED_BLOCK_STEPS 20
+
+class Step;
+
+enum StepType { propose = 1, filter, certify, finish, polling };
```

Comment:
finish & polling should be probably first_finish & second_finish ?

Reply:
use enum class if possible

Reply:
Second finish was called polling sometimes if I am not mistaken. I don't like this first&second naming, this naming is cleaner from my point of view